tests/foreman/cli/test_remoteexecution.py

```diff
@@ -290,16 +307,145 @@ def test_positive_install_multiple_packages_with_a_job_by_ip(
+        post_versions = client.run(f'rpm -q {" ".join(packages)}').stdout.splitlines()
+        assert set(pre_versions) == set(post_versions)
+        # Remove packages
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Remove Package - Katello Script Default',
+                'inputs': f'package={" ".join(packages)}',
+                'search-query': f'name ~ {client.hostname}',
+            }
+        )
+        assert_job_invocation_result(invocation_command['id'], client.hostname)
+        result = client.run(f'rpm -q {" ".join(packages)}')
+        assert result.status == len(packages)
+
+    @pytest.mark.tier3
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([8])
+    @pytest.mark.skipif(
+        (not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url'
+    )
+    def test_positive_install_remove_packagegroup_with_a_job_by_ip(
+        self, rhel_contenthost, module_org, module_ak_with_cv, target_sat
+    ):
+        """Run job to install and remove several package groups on host by ip
+
+        :id: 5be2a5c0-199a-4655-9784-e95c7ef151f5
+
+        :expectedresults: Verify the package groups were successfully installed
+            and removed on a host
+
+        :parametrized: yes
+        """
+        client = rhel_contenthost
+        groups = ['birds', 'mammals']
+        client.register(
+            module_org,
+            None,
+            module_ak_with_cv.name,
+            target_sat,
+            repo=settings.repos.yum_1.url,
+        )
+        # Install the package groups
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Install Group - Katello Script Default',
+                'inputs': f'package={" ".join(groups)}',
+                'search-query': f'name ~ {client.hostname}',
+            }
+        )
+        assert_job_invocation_result(invocation_command['id'], client.hostname)
+        result = client.run('dnf grouplist --installed')
+        assert all(item in result.stdout for item in groups)
+        # Remove one of the installed package groups
+        remove = random.choice(groups)
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Remove Group - Katello Script Default',
+                'inputs': f'package={remove}',
+                'search-query': f'name ~ {client.hostname}',
+            }
+        )
+        assert_job_invocation_result(invocation_command['id'], client.hostname)
+        result = client.run('dnf grouplist --installed')
+        assert remove not in result.stdout
+
+    @pytest.mark.tier3
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([8])
+    @pytest.mark.skipif(
+        (not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url'
+    )
+    def test_positive_install_errata_with_a_job_by_ip(
```

Comment:
Hi, those _by_ip suffixes are remainder from the past, it is no longer true that hosts connect by ip (my bad I didn't clean up the titles yet), so no need to have this information in titles and descriptions